mewarpx/mewarpx/runinfo.py

```diff
@@ -30,14 +30,14 @@ class SimInfo(object):
     This must contain:
         SimInfo.nxyz (int nx, int ny, int nz)
         SimInfo.pos_lims (floats xmin, xmax, ymin, ymax, zmin, zmax)
-        SimInfo.geom (str 'XZ', 'RZ' or 'XYZ')
+        SimInfo.geom (str 'X', 'XZ', 'RZ' or 'XYZ')
```

Comment:
Is the 1D geom string "X"? If so, should we make it consistent with warpx since, warpx uses Z for 1D?

Reply:
Haha no it is `Z`. I have no idea why I put `X` there.